### ISSUE TITLE
Buttoncombos not working: NANO USB C clone

### ISSUE THREAD

**Issue by Lo0twig** [SukkoPera/MegaDrivePlusPlus#31]
I can neither switch the region nor trigger the IGR.
I have checked the wiring several times and it seems to be correct.

The L LED stays on after you press A and goes off again after a "cooldown" the next time you press A.
The same applies to the B button.
The controller inputs do not seem to be read correctly.

If I remove all cables for the controller inputs, switching via the reset button and the reset function no longer work.

**Comment by SukkoPera** [comment]
Are you using an Arduino Nano? Make sure the fuses are set correctly.

**Comment by Lo0twig** [comment]
Yes I am using an Arduino Nano clone with usb c. I had to move the Rx led and resistor so 6button controllers work. But the behavior is the same no matter if these are removed or not.

**Comment by Lo0twig** [comment]
Uploading IMG_9630.MOV…

**Comment by SukkoPera** [comment]
Is it a clone with an ATmega328?

**Comment by Lo0twig** [comment]
Yes

**Comment by SukkoPera** [comment]
Well, I'm sorry but I don't know what to suggest. The Reset button stuff is independent of the controller snooping, so it should work even without the controller wires. Triple check the wiring and maybe post a pic of your installation so that we can see what you did.

**Comment by Lo0twig** [comment]
![NANO1](https://github.com/SukkoPera/MegaDrivePlusPlus/assets/44567649/29e659fb-f022-4ee7-a26e-a867b12dd143)
![NANO2](https://github.com/SukkoPera/MegaDrivePlusPlus/assets/44567649/daacf2c6-7c5a-4cbd-929e-ea374919b4de)

**Comment by Lo0twig** [comment]
![NANO3](https://github.com/SukkoPera/MegaDrivePlusPlus/assets/44567649/9d9906ae-9035-4914-a85c-08762a36a74c)

**Comment by Lo0twig** [comment]
![NANO4](https://github.com/SukkoPera/MegaDrivePlusPlus/assets/44567649/f5d82b04-81ff-4720-807a-1bf2bfd02ad8)

**Comment by Lo0twig** [comment]
https://github.com/SukkoPera/MegaDrivePlusPlus/assets/44567649/2ad0ee81-8f54-4ce6-9240-450361c0eb3a

**Comment by SukkoPera** [comment]
Wiring looks correct, just make sure +5V and GND are actually what they should be.

Also try another game, many games do not poll the controller at all times (in particular not during the initial screens), and since the Arduino is only snooping passively, it must wait for the game to do it.

**Comment by Lo0twig** [comment]
Pulled 5v before from an other point. No difference.
Also tried several Games via Mega EverDrive Pro. Same behavior.

**Comment by SukkoPera** [comment]
I have no more clues, sorry :(.

**Comment by Lo0twig** [comment]
😢
